.github/workflows/go.yml

```diff
@@ -0,0 +1,70 @@
+name: Go Build
+on: [push, pull_request]
```

Comment:
Need to also add for push/pullrequest to master branch

Reply:
Done.